src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -43,11 +43,13 @@ public final class JavadocTokenTypes {
      * <pre>{@code @return true if file exists}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[4x3] : [@return true if file exists]
-     *       |--RETURN_LITERAL[4x3] : [@return]
-     *       |--WS[4x10] : [ ]
-     *       |--DESCRIPTION[4x11] : [true if file exists]
-     *           |--TEXT[4x11] : [true if file exists]
+     * JAVADOC_TAG -&gt; JAVADOC_TAG
+     *    |--RETURN_LITERAL -&gt; @return
+     *    |--WS -&gt;
+     *    `--DESCRIPTION -&gt; DESCRIPTION
+     *        |--TEXT -&gt; true if file exists
+     *        |--NEWLINE -&gt; \r\n
+     *        `--TEXT -&gt;
      * }</pre>
```

Comment:
This is weird AST node. It might appear there only due to fact that it is last javadoc tag.
Please try to add one more tag after `return`, for example `see`. And share ast.
If no such node, please use AST without such node.

Reply:
Yes indeed, this is because the target javadoc tag is last of the block.
However, we can simply discard these lines and copy until line 48. I think adding another tag will not make much difference as we are trimming the AST for only the part we require ( return_literal )

The only part we need would be : 
```
JAVADOC_TAG -&gt; JAVADOC_TAG
  |--RETURN_LITERAL -&gt; @return
  |--WS -&gt;
```

Reply:
Done, swapped them so return is not the last tag.

Reply:
@SteLeo1602 , do you see any diff from the previous change, except like I said, one last line of AST trimmed ?

Reply:
I see, ``--NEWLINE ->; \r\n` seems to unrelate

Reply:
Yes, by convention, all javadoc tags begin at a new line. So, I don't see why we would require adding that to each and every example AST. Due to this, I believe dropping these would increase readability and decrease confusion among users.
Let's see if @romani agrees with me or not ;)